.github/workflows/pythonpublish.yml

```diff
@@ -13,28 +13,27 @@ on:
 
 jobs:
   deploy:
-
     runs-on: ubuntu-18.04
 
     steps:
-    - uses: actions/checkout@master
-    - name: Set up Python
-      uses: actions/setup-python@v1
-      with:
-        python-version: '3.7'
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install setuptools wheel twine pypandoc packaging
-    - name: Build and publish
-      env:
-        TWINE_USERNAME: ${{ secrets. PYPI_USERNAME }}
-        TWINE_PASSWORD: ${{ secrets. PYPI_PASSWORD }}
-      run: |
-        for v in common core features tabular multimodal text vision timeseries autogluon
-        do
-          cd "$v"/
-          python setup.py sdist bdist_wheel
-          twine upload dist/* --verbose
-          cd ..
-        done
+      - uses: actions/checkout@master
```

Comment:
We may need to avoid adding the spaces here so the PR is mainly about fixing typos.

Reply:
I have several VS Code formatting extensions enabled. I am not sure which one is causing the issue. Some extensions automatically formatted yml and markdown files on save.